nemo/collections/nlp/models/language_modeling/megatron_t5_speechlm_model.py

```diff
@@ -323,7 +324,13 @@ def load_frozen_model(self, cfg, trainer):
             t5_cfg.micro_batch_size = cfg.get('micro_batch_size', 4)
             t5_cfg.global_batch_size = cfg.get('global_batch_size', 4)
             t5_cfg.precision = trainer.precision
-            t5_cfg.tokenizer.num_sentinel_tokens = 39184 - 29056  # cfg.num_speech_tokens 39168
+            # TODO: Remove hardcoding
+            if cfg.data.get('speech_offset', 30000) > 250000:
```

Comment:
Do you know what num_sentinel_tokens is used in?

Reply:
Yes, it affects the size of the encoder and decoder embedding matrices, so it raises an error that the size of embedding matrix in the model and checkpoint are not the same.